galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -1967,7 +2005,9 @@ components:
           type: boolean
     UpdateRunStatusRequest:
       properties:
-        status:
+        runName:
+          type: string
```

Comment:
Strictly speaking we don't. The reason we went with this approach is so that if someone manually accesses the API we double check the data against the run. Realistically users will not need to, however a fail-safe is always a good idea